examples/all-clusters-app/nrfconnect/README.md

```diff
@@ -178,100 +178,58 @@ using the following command:
 
         $ git submodule update --init
```

Comment:
When applying Marcin's comments, you could also replace this with `scripts/checkout_submodules.py --shallow --platform nrfconnect` to fetch less submodules and, in turn, complete the process faster.